### PR TITLE
Upgrade docker client to 20.10.17

### PR DESCRIPTION
**Pull request by shubham149**

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -1,336 +1,35 @@
-bazil.org/fuse v0.0.0-20160811212531-371fbbdaa898/go.mod h1:Xbm+BRKSBEpa4q4hTSxohYNQpsxXPbPry4JJWOB3LB8=
```
Comment (vistaarjuneja) [inline go.sum:1]: Do we not need the `go.sum` file?

Comment (shubham149) [reply go.sum:1]: File is still present. Most of the contents of the file have been removed.

**Comment by shubham149** [comment]
It gives error `Error response from daemon: client version 1.41 is too new. Maximum supported API version is 1.40` in case of gcp windows

**Comment by tphoney** [comment]
> It gives error `Error response from daemon: client version 1.41 is too new. Maximum supported API version is 1.40` in case of gcp windows

You maybe able to use a later version of the windows image, that may allow a later client version

**Comment by shubham149** [comment]
I will try with a lower docker client and update this PR later

**Pull request closed without merging**